### ISSUE TITLE
Still unable to import x7 files?

### ISSUE THREAD

**Issue by shanzenos** [FreeDenny9700/S4_NetSphere_ResourceTool_Season1#3]
title says it all. Why did you close the issue, is it supposed to be fixed?

**Comment by FreeDenny9700** [comment]
no, for now i can't fix x7 error. I close this issue when my tool is fixed :)

**Comment by shanzenos** [comment]
Alright thanks for the heads up. Is the issue blub's library or did you confirm it's something on your end?

**Comment by FreeDenny9700** [comment]
When blub update library, i complete restool and release all on github

**Comment by shanzenos** [comment]
Have you ever tried testing previous versions of his code? I tried all of them out, but only ever used the blublib dll from them. Perhaps replacing all the dll's from a later date might work, especially since he has said it is likely to be a mis-spelling. Might seem like a stretch to try, but I don't think those libraries are going to be updated any time soon.

**Comment by FreeDenny9700** [comment]
if Blub doesn't update his library, i can't update my program. I wait Blub!!

**Comment by shanzenos** [comment]
Please update your program, blub just fixed the x7 encryption issue in his dlls. https://github.com/wtfblub/NetspherePirates/tree/master/libs

**Comment by FreeDenny9700** [comment]
Today i fix my tool and release new version of this :)

**Comment by BioAleks** [comment]
Blub create here tool. You can use it.
Just compile.

Lua encryption too fixed.

**Comment by shanzenos** [comment]
I'd still like to see this tool completely functional. (I prefer it, too.) Please complete your tool. The more the better.

**Comment by FreeDenny9700** [comment]
You can update tool yourself, download wtfblub source, conpile it, replace all dll of my project with blub's lib and enjoy.
I update my tool soon

**Comment by shanzenos** [comment]
Oh I've tried that and it still corrupts. Maybe it's because I'm using an older version. I'll try it with the newest version.

**Comment by FreeDenny9700** [comment]
Ok, today update my tool
